appveyor.yml

```diff
@@ -0,0 +1,26 @@
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - CLI_VERSION: 1.0.0-preview3-003857
+
+build_script:  
+
+  # .NET Core SDK binaries
+  - ps: $url = "https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.zip"
```

Comment:
set the url to download a specific version, is very very unstable to use `dev latest` like that.
also is not going to download preview3 atm